rpmlint/checks/FilelistCheck.py

```diff
@@ -0,0 +1,410 @@
+                    '/opt/kde3/lib64/*',
+                    '/usr/lib/pkgconfig/*',
+                    '/usr/lib/perl5/vendor_perl/5.*/*-linux-*/*',
+                ],
+                'ignorepkgif': self.notnoarch,
+            },
+            {
+                'error': 'filelist-forbidden-debuginfo',
+                'bad': [
+                    '/usr/lib/debug/*',
+                ],
+                'ignorepkgif': self.isdebuginfo,
+            },
+            {
+                'error': 'filelist-forbidden-locale',
+                'bad': [
+                    '/opt/gnome/share/locale/no',
+                    '/opt/gnome/share/locale/no/*',
+                    '/opt/kde3/share/locale/no',
+                    '/opt/kde3/share/locale/no/*',
+                    '/usr/share/locale/no',
+                    '/usr/share/locale/no/*',
+                    '/usr/share/vim/*/lang/no',
+                    '/usr/share/vim/*/lang/no/*',
+                ],
+            },
+            {
+                'error': 'filelist-forbidden-xorg',
+                'bad': [
+                    '/usr/X11R6/*',
+                ],
+                'ignorepkgif': self.isfilesystem,
+            },
+            {
+                'error': 'filelist-forbidden-suseconfig',
+                'bad': [
+                    '/sbin/conf.d/*',
+                    '/var/adm/SuSEconfig/'
+                ],
+            },
+            {
+                'error': 'filelist-forbidden-opt',
+            },
+            {
+                'error': 'filelist-forbidden-systemd-userdirs',
+                'bad': [
+                    '/etc/systemd/*',
+                    '/etc/modules-load.d/*',
+                    '/etc/tmpfiles.d/*',
+                ],
+            },
+            {
+                'error': 'filelist-forbidden-udev-userdirs',
+                'bad': [
+                    '/etc/udev/rules.d/*',
+                ],
+            },
+            {
+                'error': 'filelist-forbidden-bashcomp-userdirs',
+                'bad': [
+                    '/etc/bash_completion.d/*',
+                ],
+            },
+            {
+                'error': 'filelist-forbidden-xinetd-configuration',
+                'bad': [
+                    '/etc/xinet.d/*',
+                ],
+            },
+            {
+                'error': 'filelist-forbidden-move-to-usr',
+                'bad': [
+                    '/lib/systemd/system/*',
+                    '/lib/udev/rules.d/*',
+                ],
+            },
+            {
+                'error': 'wrong-suse-capitalisation',
+                'bad': [
+                    '*/README.SuSE',
+                ],
+                'ignorefileif': self.ghostfile,
+            },
+        ]
+
+    def check(self, pkg):
+        if pkg.isSource():
+            return
+
+        files = pkg.files()
+
+        if not files:
+            self.output.add_info('W', pkg, 'filelist-empty')
```

Comment:
This does not have a description, and I can't imagine this situation happening tbh.